src/markdown-pages/docs/cli-command-reference/runs-prepare.md

```diff
@@ -51,12 +71,25 @@ The following example creates a portfolio called `test.yaml` that contains tests
 
 All the tests in the `test.yaml` portfolio will run on the z/OS LPAR `MV2C` in the `PLEX2` cluster when the `galasactl runs submit --portfolio test.yaml` command is run, regardless of the LPAR and cluster that is specified in the CPS properties file.
 
+On Mac or Unix:
+
 ```
-galasactl runs prepare
-                        --portfolio test.yaml
-                        --stream BestSoFar
-                        --package dev.galasa.simbank.tests
-                        --override zos.default.lpar=MV2C
+galasactl runs prepare \
+                        --portfolio test.yaml \
+                        --stream BestSoFar \
+                        --package dev.galasa.simbank.tests \
+                        --override zos.default.lpar=MV2C \
+                        --override zos.default.cluster=PLEX2
+```
+
+On Windows:
+
+```
+galasactl runs prepare `
+                        --portfolio test.yaml `
+                        --stream BestSoFar `
+                        --package dev.galasa.simbank.tests `
+                        --override zos.default.lpar=MV2C `
```

Comment:
same as above. MV2C should be changed...etc.

Reply:
updated